bundle/src/main/java/io/wcm/caconfig/editor/RootPathProvider.java

```diff
@@ -25,22 +25,23 @@
 import org.osgi.annotation.versioning.ConsumerType;
 
 /**
- * Allows to provide a dynamic root path for path browser widget.
+ * Allows to provide a dynamic root path for path or tag browser widget.
  * Implement as OSGi service and set a service property {@link #PROPERTY_SELECTOR} which is used to reference
  * the implementation in the context-aware configuration property metadata options.
  */
 @ConsumerType
-public interface PathBrowserRootPathProvider {
+public interface RootPathProvider {
```

Comment:
this is a breaking change to the existing API, that's why the package version would have to been raised to 2.0. we should avoid this for all cost for backward compatibility.

we can either reuse the PathBrowserRootPathProvider  also for the tag browser, or introduce a new interface for tag browser.

Reply:
New provider and service introduced for tagbrowser.  They both have their own implementation now (+ tests).  Also changed the Version back to 1.6 (from 2.0).